lecture_notes/09 functions/2-scope.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+let cities: [String] = ["Athens", "Roma", "London", "Beijing", "Kiev", "Riga"]
+let citiesLengths: [Int] = cities.map {$0.count}
```

Comment:
You have t choices
- Improve your understanding of swift syntax
- Rewrite this line to well known constructions like for

Reply:
```swift
var citiesLengths: [Int] = []

func cityLength() {
    for city in cities {
        let cityLength = city.count
        citiesLengths.append(cityLength)
    }

}
```